v1/openapi/schemas/search-api-request.yaml

```diff
@@ -0,0 +1,21 @@
+x-component: 'schemas/search-api-request'
```

Comment:
It seems you forget to delete this file after rename api-request to original-request. Also, sometimes in other schemas you still use api-request. Its really confusing, I think you should use same name everywhere.

Reply:
very helpful comment. I fix it